apps/molgenis-components/src/components/forms/formUtils/formUtils.ts

```diff
@@ -40,7 +40,7 @@ function getColumnError(
     return undefined;
   }
   if (column.required) {
-    if (column.required === "true") {
+    if (column.required === "true" || column.required === true) {
```

Comment:
if this also needs to check the for uppercase TRUE, it is worth it to create a "isRequired(column.required)" function